### ISSUE TITLE
[ACCESSIBILITY] Help panel close button is missing text

### ISSUE THREAD

**Issue by laurendoone** [Hyperobjekt/seda-map#543]
When you open the Help panel on the Explorer, the close button is empty or missing text.

- What It Means: A button is empty or has no value text.
- Why It Matters: When navigating to a button, descriptive text must be presented to screen reader users to indicate the function of the button.
- What To Do: Place text content within the element or give the element a value attribute.

![image](https://user-images.githubusercontent.com/76966798/106820375-131c0980-6630-11eb-9899-4efca00fc64a.png)
